gherkin-utils/java/gherkin-utils/src/main/java/gherkin/utils/formatter/model/BackgroundStepContainer.java

```diff
@@ -0,0 +1,54 @@
+package gherkin.utils.formatter.model;
+
+import io.cucumber.messages.Messages;
+
+import java.util.List;
+
+public class BackgroundStepContainer implements IStepContainer {
```

Comment:
All classes that are not part of the public API should be private or package private.

Reply:
Sr, this is my mistake. I have already fixed.